tests/recipes/test_full_finetune_distributed.py

```diff
@@ -58,22 +72,23 @@ def test_loss(self, tmpdir, monkeypatch):
 
         cmd = f"""
         tune run --nnodes 1 --nproc_per_node 2 full_finetune_distributed \
-            --config llama2/7B_full \
+            --config {config} \
             output_dir={tmpdir} \
-            checkpointer._component_=torchtune.utils.FullModelHFCheckpointer \
+            checkpointer._component_={ckpt_component} \
             checkpointer.checkpoint_dir='{ckpt_dir}' \
             checkpointer.checkpoint_files=[{ckpt_path}]\
             checkpointer.output_dir={tmpdir} \
-            checkpointer.model_type=LLAMA2 \
+            checkpointer.model_type={model_type} \
```

Comment:
Actually can you check if we even need all caps for model type here? Since ModelType enum maps to the lower-case string anyways. (this is related to my other comment)